handlers.py

```diff
@@ -28,6 +29,35 @@ def split_by_first_mention(message):
         return '', msg
 
 
+def message_starts_with_any_fragment(message, fragments):
+    return any(message.lower().startswith(query_game_fragment.lower())
+               for query_game_fragment in fragments)
+
+
+def message_pertains_to_all_games(message):
+    query_game_fragments = ['games', 'game', 'list', 'g']
+    return message_starts_with_any_fragment(message, query_game_fragments)
+
+
+def get_game_name_or_alias_from_message(message):
+    if lookup_game_by_name_or_alias(message):
+        game = lookup_game_by_name_or_alias(message)
+        for potential_name in [game.name] + game.aliases:
+            if message.lower().startswith(potential_name.lower()):
+                return potential_name
+
+
+def default_game_json_template(game_name):
```

Comment:
Maybe this should be called `get_default_game_dict_representation` since it's definitely not json, and it's not a template in the sense that we replace placeholder values in it - we actually just write keys.